.github/workflows/release.yml

```diff
@@ -131,13 +150,39 @@ jobs:
           echo "$promotion_summary :frog: Version ${{ inputs.release_version }} was promoted to ${rt_repo}" >> $GITHUB_STEP_SUMMARY
 
       - name: Checkout reusable workflow source repo
-        if: ${{ inputs.box_upload == true }}
         uses: actions/checkout@v4
         with:
           repository: pentaho/actions-common
           path: actions-common
-          ref: stable
+          ref: HNC-539
```

Comment:
Don't forget to rollback this to `stable`

Reply:
yes

Reply:
Check on line:155 before merging

Reply:
yes

Reply:
approved